src/Posts.tsx

```diff
@@ -65,8 +229,64 @@ class Posts extends React.Component<Props, State> {
           <div key={post.id} className="posts">
             <h5 className="post-topic">{post.topic}</h5>
             <h6 className="post-post">{post.post}</h6>
+            <Moment format="HH:mm DD.MM.YYYY">{post.created.toString()}</Moment>
           </div>
         ))}
+        <form>
```

Comment:
Maybe move the form on top of the list so the form does not get hidden with long list of posts? :D